src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+
+> Therefore the earlier determinations which (on the _pathway to truth_) _existed for themselves_, as for instance, that of subjective and objective, or also of thought and being, of concept and reality, no matter from what standpoint they were determined, are _now in their truth_, that is, in their unity, _reduced to forms_. In their difference they therefore implicitly remain, _in themselves_, the whole concept, and this concept is posited in the division only under its own determinations (Hegel, 2010, 39/21.45).
+
+_The objective logic_ concerns the determinations or concepts that are typically understood to exist immediately, such as `being`, `existence`, `quality`, `the one`, `the many` and so forth. In more technical terms, It is the concept _implicitly in itself_.
+
+_The subjective logic_, on the other hand, concerns explicitly conceptual determinations, such as `universality`, `particularity`, `singularity`, `judgment`, `inference`, `means`, `purpose`, `cognition` and many more. This is the concept _that exists for itself_.
+
+Now, this is essentially the first broad division of logic, but, as Hegel notes, being and the concept do not stand wholly apart from each other but are related. Indeed, their very difference logically establishes a link between the two. This reference constitutes a particular logic unto itself that is known as the sphere of mediation or _The Doctrine of Essence_. As Hegel elaborates:
+
+> [The sphere of mediation is] the concept as a system of _reflected determinations_, that is, of being as it passes over into the in-itselfness of the concept &ndash; a concept which is in this way not yet posited for itself _as such_ but also fettered by an immediate being still external to it (Hegel, 2010, 40/21.46).
+
+The _difference_ between being and thought necessitates a third element between them and so it is more precise to divide the logic into three doctrines rather than two parts. Hegel also notes that with essence the stalwart objectivity of being is progressively denigrated to include the inwardness of the concept; however, precisely for that very reason, the concept is unable to be made fully explicit while the immediacy of being persists. Essence is therefore the purgation of the concept, as it were, since the concept is actively failing to be the concept proper while being remains something immediately external to it and so must overcome these failures by the discovery of a logic that is not bound by mere difference. 
+
+## Further Remarks
+
+### On the Term Element
+
+Hegel highlights the term _element_ when discussing the unity of the doctrines. As George di Giovanni notes in his translation, element has classical connotations harkening back to the four elements of "air", "water", "earth" and "fire" where it is understood that something has an underlying nature of its own _but also_ has implied  modes of being or relationships to other elements (Hegel, 2010, 39). In Aristotle, we find element defined as: 
```

Comment:
This theme of the "element" has come a bit out of nowhere and is a bit confusing. I'm not convinced it's the most helpful thing to include in an introductory essay on the division of the Logic. But if you want to keep it, I would suggest beginning with a quotation for Hegel and then move on to unpack what it might mean.
Personally, however, I think it's the kind of thing that should not be in the opening sections of an introductory section as it is too niche of a comment.

Reply:
It's put under "Further Remarks", so that kind of flags it as optional. Right now, it's too small to move to its own file and it is part of this piece of text. Maybe if we do something on "element" later we can shift it.